api/database/migrations/20191114212136-teams.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface
+      .createTable("Teams", {
+        id: {
+          allowNull: false,
+          type: Sequelize.INTEGER,
+          autoIncrement: true,
+          primaryKey: true
+        },
+        name: {
+          type: Sequelize.STRING(150),
+          allowNull: false
+        },
+        teamCode: {
+          type: Sequelize.STRING(4),
+          allowNull: false,
+          unique: true
+        },
+        ownerId: {
+          type: Sequelize.STRING,
+          references: {
+            model: "HackerProfiles",
+            key: "userId"
+          }
+        }
+      })
+      .then(() => {
```

Comment:
need to chain promises, otherwise might run into some interesting race condition where we try to add a teamId column before the Team table is created